FitVid/src/Components/Video/Video.js

```diff
@@ -1,26 +1,37 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import {
   FiMoreVertical,
   MdWatchLater,
   RiPlayListAddFill,
   AiFillLike,
   AiOutlineLike,
+  MdOutlineWatchLater,
 } from "../../Assets/icons";
```

Comment:
Suggestion : Check the naming for icons

Reply:
I will check it later when i will optimize my app